tools/bazar/fields/MapField.php

```diff
@@ -25,7 +25,26 @@ public function __construct(array $values, ContainerInterface $services)
         $this->longitudeField = $values[self::FIELD_LONGITUDE_FIELD] ?? 'bf_longitude';
         $this->autocomplete = $values[self::FIELD_AUTOCOMPLETE];
 
-        $this->propertyName = 'carte_google';
+        $this->propertyName = 'geolocation';
```

Comment:
```suggestion
        $this->propertyName = 'geolocation';
        $this->label = $this->propertyName;
```

C'est pour simplifier la vie des export/import